File: AdoptaMe_Spring/AdoptaMe/src/main/java/mx/edu/unpa/apidroid/dtos/responses/UsuarioResponse.java
package mx.edu.unpa.apidroid.dtos.responses;

import lombok.Builder;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@Builder
public class UsuarioResponse {
    private Long id;
    private String nombre;
    private String apellidoPaterno;
    private String apellidoMaterno;
    private String email;
    private String foto;
    private String telefono;
    private Boolean activo;
    private LocalDateTime fechaRegistro;
}

File: AdoptaMe_Android/MiAndroid/app/src/main/java/mx/edu/unpa/miandroid/model/UsuarioResponse.kt
package mx.edu.unpa.miandroid.model

import java.util.Date

data class UsuarioResponse(
    val id: Long,
    val nombre: String,
    val apellidoPaterno: String,
    val apellidoMaterno: String? = null,
    val email: String,
    val telefono: String? = null,
    val activo: Boolean? = true,
    val fechaRegistro: String? = null
)